test/Azure.Functions.Cli.Tests/ActionsTests/StartHostActionTests.cs

```diff
@@ -186,6 +190,53 @@ private IFileSystem GetFakeFileSystem(IEnumerable<(string folder, string functio
             return fileSystem;
         }
 
+        [Theory]
+        // In-proc target, in-proc 8 argument, project configured for .NET 8. Succeeds.
+        [InlineData(WorkerRuntime.dotnet, DotnetConstants.InProc8HostRuntime, true, false)]
+        // In-proc target, in-proc 8 argument, project NOT configured for .NET 8. Fails.
+        [InlineData(WorkerRuntime.dotnet, DotnetConstants.InProc8HostRuntime, false, true)]
+        // In-proc target, in-proc 6 argument, project NOT configured for .NET 8. Succeeds.
+        [InlineData(WorkerRuntime.dotnet, DotnetConstants.InProc6HostRuntime, false, false)]
+        // In-proc target,'default' argument, project configured for .NET 8. Fails.
+        [InlineData(WorkerRuntime.dotnet, "default", true, true)]
+        // Isolated target,'default' argument, project NOT configured for .NET 8. Succeeds.
+        [InlineData(WorkerRuntime.dotnetIsolated, "default", true, false)]
+        // Isolated target,'default' argument, project configured for .NET 8. Succeeds.
+        [InlineData(WorkerRuntime.dotnetIsolated, "default", false, false)]
+        // Isolated target,in-proc 8 argument, project configured for .NET 8. Fails.
+        [InlineData(WorkerRuntime.dotnetIsolated, DotnetConstants.InProc8HostRuntime, true, true)]
+        // Isolated target,in-proc 6 argument, project not configured for .NET 8. Fails.
+        [InlineData(WorkerRuntime.dotnetIsolated, DotnetConstants.InProc6HostRuntime, false, true)]
+        // Unsupported runtime targets
+        [InlineData(WorkerRuntime.dotnetIsolated, "somevalue", false, true)]
+        [InlineData(WorkerRuntime.dotnet, "somevalue", false, true)]
+        public async Task ValidateHostRuntimeAsync_MatchesExpectedResults(WorkerRuntime currentRuntime, string hostRuntimeArgument, bool validNet8Configuration, bool expectException)
```

Comment:
Given the default branch of the switch statement I commented on separately, should there also be test for some other worker runtime?